tsfc/fem.py

```diff
@@ -233,6 +234,11 @@ def flat_index(ordered_deriv):
         return result[0]
 
 
+def zero_tracking_literal(table, ufl_element):
+    track_zeros = isinstance(ufl_element, (ufl.EnrichedElement, ufl.MixedElement))
```

Comment:
@FabioLuporini Yes, I understand that.  But I think your mention of derivatives before was *misleading*, perhaps?  You're not *not interested* in derivatives producing zero columns... if derivatives made the first 8 columns zero then you'd like to exploit that, I'm sure?

The thing you're not interested in is knocking out single columns, for the reason you posted, therefore you don't do exhaustive table-sniffing.  Right?